embedded-aerospike/src/main/java/com/playtika/test/aerospike/EmbeddedAerospikeBootstrapConfiguration.java

```diff
@@ -83,7 +83,13 @@ public GenericContainer aerospike(AerospikeWaitStrategy aerospikeWaitStrategy,
                         .withEnv("STORAGE_GB", String.valueOf(1))
                         .withCreateContainerCmdModifier(cmd -> cmd.getHostConfig().withCapAdd(Capability.NET_ADMIN))
                         .waitingFor(waitStrategy);
-
+        String featureKey = properties.featureKey;
+        if (featureKey != null) {
+            // see https://github.com/aerospike/aerospike-server-enterprise.docker/blob/master/aerospike.template.conf
+            aerospike
```

Comment:
FYI: you can use this feature without this change, just by setting container env properties, although I feel like it should be more convenient with this change.

Reply:
Right. But this is much more convenient and transparent.